### ISSUE TITLE
Meshing of tessellations cut by ecylinder

### ISSUE THREAD

**Issue by Sofia-Y** [rquey/neper#124]
This example works just fine in 2D for both "quad" and "tri" elements. Also, worked for 3D "hex" elements but not for the "tet" elements. 

Any help, please?

```
$ neper -T -n 1111 -id 1 -oricrysym cubic -ori uniform -format tess -transform "cut(ecylinder(0,0.5,0,0,0,1,1,0,0,0,1,0,0.5,0.05))" 
$ neper -M n1111-id1.tess -elt tet -format 'inp' 

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.1.2
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt
Info   : Running on 32 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2020, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/sy422/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -M loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] n1111-id1.tess -elt tet -format inp
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Loading input data...
Info   :   - Loading tessellation...
Info   :     [i] Parsing file `n1111-id1.tess'...
Info   :     [i] Parsed file `n1111-id1.tess'.
Info   : Meshing...
Info   :   - Preparing... (cl = 0.04857) 100%
Info   :   - 0D meshing... 100%
Info   :   - 1D meshing... 100%
Info   :   - 2D meshing...   2% (0.082|0.63/98%| 2%| 0%)
node not caught during backward projection
Error  : You have discovered a bug in Neper!  Please file an issue at
         https://github.com/rquey/neper/issues.  Thank you.

Aborted
```

**Comment by rquey** [comment]
It is a bit disappointing, but as it turns out, `-transform cut(ecylinder(...))` will generate the tessellation just fine, but meshing is not supported yet...

**Comment by rquey** [comment]
Fixed in 4.4.1.
